terratest/test/go.mod

```diff
@@ -0,0 +1,10 @@
+module testterratest
+
+go 1.14
+
+require (
+	github.com/gruntwork-io/terratest v0.26.1
+	github.com/stretchr/testify v1.5.1
+	k8s.io/api v0.17.0
```

Comment:
- terratest .mod file has same modules but different versions than main .mod file. Please unify.

Reply:
done

Reply:
have to fix to 0.19.3 as terratest doesn't work with 0.20.2 yet